videoframegrid.py

```diff
@@ -68,6 +71,28 @@ def swap_frame(self, frame):
             else:
                 self.selected_frame = frame
 
+    def toggle_fullscreen(self, selected_frame):
+        if not self.fullscreen:
+            for videoframe in self.videoframes:
```

Comment:
I think list comprehensions should be used for simpler things. I think it's good as it is now.